README.md

```diff
@@ -69,3 +69,12 @@ PR title: [docs] Changed this and that
 ```
 
 All PRs will require at least one review from CSCH staff. If it's been over a week, feel free to ping us once in the Discord.
+
+## License & Copyright
+
+
+[![License: MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
```

Comment:
Please just place this under the main header, and no `License & Copyright` subheader or `megabot, cscareerbot` part.

The shield just should reference the repo, not be fixed on MIT (so would need to reference https://shields.io/category/license ). It may appear as "unknown" but that is valid until it is merged.